extra-files/Conect/conect_tool_list.yml

```diff
@@ -1269,3 +1269,33 @@ tools:
   tool_panel_section_id: None
   tool_panel_section_label: None
   tool_shed_url: toolshed.g2.bx.psu.edu
+- name: arriba
+  owner: jjohnson
+  tool_panel_section_id: genefusions
+  tool_panel_section_label: Gene fusions
+  tool_shed_url: toolshed.g2.bx.psu.edu
+- name: arriba_download_reference
+  owner: jjohnson
+  tool_panel_section_id: genefusions
+  tool_panel_section_label: Gene fusions
+  tool_shed_url: toolshed.g2.bx.psu.edu
+- name: arriba_draw_fusions
+  owner: jjohnson
+  tool_panel_section_id: genefusions
+  tool_panel_section_label: Gene fusions
+  tool_shed_url: toolshed.g2.bx.psu.edu
+- name: arriba_get_filters
+  owner: jjohnson
+  tool_panel_section_id: genefusions
+  tool_panel_section_label: Gene fusions
+  tool_shed_url: toolshed.g2.bx.psu.edu
+- name: star_fusion
+  owner: iuc
+  tool_panel_section_id: genefusions
+  tool_panel_section_label: Gene fusions
+  tool_shed_url: toolshed.g2.bx.psu.edu
+- name: data_manager_star_index_builder
```

Comment:
Note @NairaNaouar  que tu rencontreras possiblement le bug de mémoire de start index builder quand tu construiras ton index hg38. Dis moi si cela t'arrive, je sais comment hacker le truc (c'est fait dans artbio)

Reply:
Merci @drosofff je viens de le lancer et je vais voir ce que ça donne et je te dis

Reply:
Tu avais raison, ça n'est pas passé et j'ai l'erreur suivante : 

` EXITING because of FATAL PARAMETER ERROR: limitGenomeGenerateRAM=4820000000is too small for your genome
SOLUTION: please specify --limitGenomeGenerateRAM not less than 8746522208 and make that much RAM available 
Apr 27 13:58:31 ...... FATAL ERROR, exiting `
Du coup, je veux bien ton input pour "hacker" le truc @drosofff

Reply:
J'ai rajouté la ligne
```
GALAXY_MEMORY_BYTES=100550893493 ;
```
dans le fichier 
```
~/shed_tools/toolshed.g2.bx.psu.edu/repos/iuc/data_manager_star_index_builder/c520a52b5174/data_manager_star_index_builder/data_manager/rna_star_index_builder.xml
```
Ca devrait passer maintenant @NairaNaouar

Reply:
Heu oui, tu as simplement zappé qu'il faut redémarrer le serveur car les xml sont chargé en mémoire au démarrage... (je pense)

Reply:
Indeed @drosofff après redémarrage c'est lancé avec les bons paramètres 😄 
Merci